src/include/distributed/connection_management.h

```diff
@@ -187,6 +198,9 @@ typedef struct MultiConnection
 	/* number of bytes sent to PQputCopyData() since last flush */
 	uint64 copyBytesWrittenSinceLastFlush;
 
+	/* replication option */
+	bool requiresReplicationOption;
```

Comment:
nitpick: Drop the 'option' just requiresReplication or requires requiresReplicationParam.

Reply:
done.